### PR TITLE
Migrate project to yarn v1

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

This PR migrates the project back to yarn v1.

### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/20138

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]
Seems to work as expected

**Comment by sleshchenko** [comment]
[dashboard-ci-test]

File: .deps/problems.md [hunk 1]
```diff
@@ -2,18 +2,20 @@
 
 ## UNRESOLVED Production dependencies
 
-1. `@patternfly/react-table@4.29.0`
-2. `immutable@4.0.0-rc.12`
+1. `@eclipse-che/devworkspace-client@0.0.1-1625495079`
```
Comment (olexii4) [inline .deps/problems.md:5]: should be fixed in https://github.com/eclipse-che/che-dashboard/blame/main/.deps/EXCLUDED/prod.md#L6

**Review by olexii4 — APPROVED** [review]
LGTM

File: .deps/problems.md (hunk 1 above)
Comment (akurinnoy) [reply .deps/problems.md:5]: this isn't a critical issue so let's provide the fix in one of the following PRs

**Pull request merged**